docker/Dockerfile-scripted

```diff
@@ -0,0 +1,68 @@
+# This is based on previous work in https://github.com/nilsnolde/docker-valhalla
+
+# Take the plain valhalla docker image,
+# remove a few superfluous things and
+# create a new runner image from ubuntu:24.04
+# with the previous runner's artifacts
+ARG VALHALLA_BUILDER_IMAGE=ghcr.io/valhalla/valhalla:latest
+FROM $VALHALLA_BUILDER_IMAGE AS builder
+LABEL org.opencontainers.image.authors="nilsnolde+github@proton.me"
+
+# remove some stuff from the original image
```

Comment:
this is just some small space optimization, we don't need all the debugging scripts here IMO